request/src/main/java/com/vimeo/networking2/VimeoApiClient.kt

```diff
@@ -82,6 +83,7 @@ interface VimeoApiClient {
      * @return A [VimeoRequest] object to cancel API requests.
      */
     fun createAlbum(
+        uri: String,
```

Comment:
Now that this function takes a URI, we should create a counterpart that accepts a `User` as well. This will make it easier for consumers to create an album, without needing to select the correct connection.